### PR TITLE
Feature: Implemented lic12 /issue #10

### PR DESCRIPTION
**Pull request by Darpos**
Get compile errors with accessing fields through parameters, but worked previously and it's been used elsewhere in code so it should be fine.

### CODE REVIEW

**Review by andersblomqvist — CHANGES_REQUESTED** [review]
We decided upon using `-1 = NOTUSED`, `0 = ORR` and `1 = ANDD`. All the LICs should still return a boolean value.

Additionally, every new feature should also contain a test. Change the return type to a boolean and add a test and this will be fine.

File: src/junit/LIC_test.java [hunk 1]
```diff
@@ -8,4 +8,16 @@
 import static org.junit.Assert.assertTrue;
 
 public class LIC_test {
+    /*
```
Comment (andersblomqvist) [inline src/junit/LIC_test.java:11]: From the assessment criteria on [Canvas](https://canvas.kth.se/courses/37918/assignments/235345): 

* The contract that is tested is clearly formulated in natural language
* The assertions of the tests correctly and completely assess the contract

I think the natural language part is missing, which should define what the test is testing and expectation.

**Review by andersblomqvist — CHANGES_REQUESTED** [review]

**Comment by Darpos** [comment]
Issues with the file don't accept

**Comment by Darpos** [comment]
> Issues with the file don't accept

My test don't run on this branch, trying to fix so don't accept

File: src/junit/LIC_test.java [hunk 2]
```diff
@@ -23,14 +22,14 @@ public void cmv_getter_test(){
     public void checkIfValidGivesTrue(){
         int[][] datapoints = {{4,0},{3,0},{2,0},{1,0},{0,0}};
         CMV cmv = new CMV(datapoints);
-        assertTrue(cmv.get_cmv_value(5));
+        assertTrue(cmv.lic5_calculate());
```
Comment (andersblomqvist) [inline src/junit/LIC_test.java:25]: This causes an error due to function is private. There must been a mistake by removing the functioning code `cmv.get_cmv_value(5)`.

File: src/junit/LIC_test.java [hunk 3]
```diff
@@ -23,14 +22,14 @@ public void cmv_getter_test(){
...
     }
     //LIC 5
     @Test
     public void checkIfInvalidGivesFalse(){
         int[][] datapoints = {{0,0},{1,0},{2,0},{3,0},{4,0}};
         CMV cmv = new CMV(datapoints);
-        assertFalse(cmv.get_cmv_value(5));
+        assertFalse(cmv.lic5_calculate());
```
Comment (andersblomqvist) [inline src/junit/LIC_test.java:32]: Here as well.

**Review by andersblomqvist — CHANGES_REQUESTED** [review]

**Review by andersblomqvist — APPROVED** [review]
👍

**Pull request merged**